src/mono/nuget/Microsoft.NET.Sdk.Mono.Toolchain.Manifest/Microsoft.NET.Sdk.Mono.Toolchain.Manifest.pkgproj

```diff
@@ -0,0 +1,31 @@
+<Project DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
+
+  <PropertyGroup>
+    <PackageDescription>Internal toolchain package not meant for direct consumption. Please do not reference directly.</PackageDescription>
```

Comment:
Consider setting `<UseRuntimePackageDisclaimer>true</UseRuntimePackageDisclaimer>` which adds the standard description for internal packages that we use everywhere else: https://github.com/dotnet/runtime/blob/cd271c1437445e41b31ebfacb9e63f6f9d54ab61/Directory.Build.targets#L54.